packages/entity-server/src/app/createApp.ts

```diff
@@ -9,14 +9,14 @@ import bodyParser from 'body-parser';
 import errorHandler from 'api-error-handler';
 
 import { Authenticator } from '@tupaia/auth';
-import { ModelRegistry } from '@tupaia/database';
+import { EntityServerModelRegistry } from '../types';
 
 import { addRoutesToApp } from './addRoutesToApp';
 
 /**
  * Set up express server with middleware,
  */
-export function createApp(models: ModelRegistry) {
+export function createApp(models: EntityServerModelRegistry) {
```

Comment:
Ah good point, I think you're right I'll change that 👍

Reply:
In the end I went with leaving the full registry on as we need at least the `Project` model as well as the `Entity` model, and I thought it might be nice to have access to all the utility methods on the `ModelRegistry` class (tho I'm not sure if we'll be actually using them...)

Reply:
Sweet, fair enough!